doc/reference/storage_lvm.md

```diff
@@ -1,45 +1,62 @@
 (storage-lvm)=
 # LVM - `lvm`
 
- - Uses LVs for images, then LV snapshots for instances and instance snapshots.
- - The filesystem used for the LVs is ext4 (can be configured to use xfs instead).
- - By default, all LVM storage pools use an LVM thinpool in which logical
-   volumes for all LXD storage entities (images, instances, etc.) are created.
-   This behavior can be changed by setting "lvm.use\_thinpool" to "false". In
-   this case, LXD will use normal logical volumes for all non-instance
-   snapshot storage entities (images, instances, etc.). This means most storage
-   operations will need to fallback to rsyncing since non-thinpool logical
-   volumes do not support snapshots of snapshots. Note that this entails
-   serious performance impacts for the LVM driver causing it to be close to the
-   fallback DIR driver both in speed and storage usage. This option should only
-   be chosen if the use-case renders it necessary.
- - For environments with high instance turn over (e.g continuous integration)
-   it may be important to tweak the archival `retain_min` and `retain_days`
-   settings in `/etc/lvm/lvm.conf` to avoid slowdowns when interacting with
-   LXD.
-
-## Storage pool configuration
+{abbr}`LVM (Logical Volume Manager)` is a storage management framework rather than a file system.
+It is used to manage physical storage devices, allowing you to create a number of logical storage volumes that use and virtualize the underlying physical storage devices.
+
+Note that it is possible to over-commit the physical storage in the process, to allow flexibility for scenarios where not all available storage is in use at the same time.
+
+## Terminology
+
+LVM can combine several physical storage devices into a *volume group*.
+You can then allocate *logical volumes* of different types from this volume group.
+
+One supported volume type is a *thin pool*, which allows over-committing the resources by creating volumes that are larger than the available physical storage.
+Another type is a *volume snapshot*, which captures a specific state of a logical volume.
+
+## `lvm` driver in LXD
+
+The `lvm` driver in LXD uses logical volumes for images, and volume snapshots for instances and snapshots.
+The file system used for the logical volumes is ext4 by default.
+You can configure the `lvm` driver to use XFS instead by setting the {ref}`block.filesystem <storage-lvm-vol-config>` configuration for the volume (or the corresponding `volume.block.filesystem` configuration on the storage pool for all volumes in the pool).
```

Comment:
It feels like we should have a generic section about what values are supported by the `block.filesystem` (and by association the `volume.block.filesystem`) setting for all supported storage pools, rather than duplicate that. 

Because they are the same independent of storage pool type (for those storage pools that support the `block.filesystem` setting), they are: btrfs, ext4 and xfs. Defaulting to ext4.

Reply:
The option is only listed for LVM and Ceph RBD, or is it missing somewhere?
If it's only those two, I think it's fine to mention it in both places (since there is a limitation for XFS for Ceph RBD). But I'll also add the options and default to the table.

Reply:
Yes its only relevant for LVM and Ceph. I'm not sure what you mean about the XFS limitation for Ceph RBD, here we are talking about the filesystems used on the Ceph RBD (block volumes) rather than the filesystem used to host Ceph itself. I've checked the code and both LVM and Ceph drivers allow the same set of filesystems.

Reply:
Ah okay - yes, I confused those two. Then we can take it out here as well.